pkg/settings/settings.go

```diff
@@ -308,6 +311,37 @@ func (settings *Settings) ShouldInitialism() bool {
 	return !settings.NoInitialism
 }
 
+// HasCustomRename returns if the setting to rename columns with
+// custom rename rules is set.
+func (settings *Settings) HasCustomRename() bool {
+	return settings.CustomColumnRename != ""
+}
+
+const multiSplitter = ","
+const ruleSplitter = ":"
+
+// ParseCustomRenameRules reads a custom rename setting string
+// and returns a map for easier rule access.
+// customRenameRule example: column1:Column2,col10:column10.
+// multiple rules are split with comma
+func (settings *Settings) ParseCustomRenameRules() map[string]string {
+	if !settings.HasCustomRename() {
+		return nil
+	}
+
+	m := make(map[string]string, strings.Count(settings.CustomColumnRename, multiSplitter)+1)
+	for _, rule := range strings.Split(settings.CustomColumnRename, multiSplitter) {
```

Comment:
Still thinking about runtime performance here: lets split once and use the length of the rules:

```go
rules := strings.Split(settings.CustomColumnRename, rulesSeparator)
m := make(map[string]string, len(rules))
for ...
```